clothcalc_placeholder.js

```diff
@@ -7799,13 +7807,26 @@
                         eval("MarketWindow.getClearName = " + save.MarketWindow)
                     }
                 };
+
+                _self.injectGetBids = function() {
+		    if (typeof save.MarketWindowTab == "undefined") {
```

Comment:
But if I take the "apply" method, the other two eval functions for MarketWindow.showTab at line 7900 won't work anymore, because MarketWindow.showTab would be a new function.